src/goMain.ts

```diff
@@ -57,8 +57,16 @@ export function activate(ctx: vscode.ExtensionContext): void {
 
 	updateGoPathGoRootFromConfig().then(() => {
 		const updateToolsCmdText = 'Update tools';
-		const prevGoroot = ctx.globalState.get('goroot');
-		const currentGoroot = process.env['GOROOT'];
+		interface GoInfo {
+			goroot: string;
+			version: string;
+		}
+		const toolsGopath = getToolsGopath() || getCurrentGoPath();
+		let toolsGoInfo: { [id: string]: GoInfo; } = ctx.globalState.get('toolsGoInfo') || {};
+		if (!toolsGoInfo[toolsGopath]) toolsGoInfo[toolsGopath] = { goroot: null, version: null };
+		const prevGoroot = toolsGoInfo[toolsGopath].goroot;
+		const currentGoroot: string = process.env['GOROOT'];
+		vscode.window.showInformationMessage(currentGoroot);
```

Comment:
Is this info message needed? This will show up everytime the extension activates which is for every new Go project opened.